salt/fileclient.py

```diff
@@ -813,10 +813,10 @@ def hash_file(self, path, saltenv='base'):
         ret = {}
         try:
             path = self._check_proto(path)
-        except MinionError:
+        except MinionError, err:
```

Comment:
Oof! I've been doing the comma thing for years, but just looked this up as you pointed it out, and it seems to be a legacy thing, a holdover from Python <= 2.5. I'll fix it, thanks!